### ISSUE TITLE
Add option to disable the transparent notifications that drop down from the window

### ISSUE THREAD

**Issue by tommack** [git-up/GitUp#328]
These notifications never tell me anything interesting and always get in the way of things I want to click on or see. I know what branch I'm on as it is clearly displayed in the main window and the warning about new commits being available appears to be wrong most of the time.

**Comment by kud** [comment]
I feel the same here.

**Comment by sesam** [comment]
Yep, I've also seen the New commits notice show up when in fact there was nothing to be seen.
Would you want to have this feature if it was 99.9% reliable and maybe also would show up in a less intrusive way, so maybe bottom of screen and only in map view and it waits for you to stop doing other things before showing up?

-> Otherwise it may be best to just remove this notice.
